addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanRuleUnitTest.java

```diff
@@ -100,6 +102,14 @@ void shouldReturnExpectedMappings() {
                 is(equalTo(CommonAlertTag.WSTG_V42_ATHN_06_CACHE_WEAKNESS.getValue())));
     }
 
+    @Test
+    void shouldReturnExampleAlerts() {
```

Comment:
Since this is now generically tested could be exposed a method for their number/risk and a method to assert them further.